examples/tutorials/api/nested_sampling_Crab.py

```diff
@@ -0,0 +1,399 @@
+"""
+Bayesian analysis with nested sampling
+======================================
+
+Bayesian analysis with nested sampling.
+
+"""
+
+
+######################################################################
+# Context
+# -------
+#
+# 1. Bayesian analysis
+# ~~~~~~~~~~~~~~~~~
+#
+# Bayesian inference uses prior knowledge, in the form of a prior
+# distribution, in order to estimate posterior probabilities which we
+# traditionally visualise in the form of corner plots. These distributions
+# contain much more information than a single best-fit as they reveal not
+# only the “best model” but the (not always Gaussian) errors and
+# correlation between parameters.
```

Comment:
A maximum likelihood fit would contain this information in the Hessian (see e.g. iminuit) for Gaussian degeneracies. non-Gaussian degeneracies are complex to handle with maximum likelihood.

Reply:
You can also look at the method section of https://ui.adsabs.harvard.edu/abs/2014A%26A...564A.125B/abstract maybe there are some useful things in there.